lib/task_helpers.rb

```diff
@@ -54,12 +54,11 @@ def self.catchup_index(start_time, index_name, batch_size=100, sleep_time=0)
     def self.create_index(name=nil)
       name ||= "#{Content::ES_INDEX_NAME}_#{Time.now.strftime('%Y%m%d%H%M%S%L')}"
 
-      mappings = {}
-      [Comment, CommentThread].each do |model|
-        mappings.merge! model.mappings.to_hash
-      end
+      Elasticsearch::Model.client.indices.create(index: name)
 
-      Elasticsearch::Model.client.indices.create(index: name, body: {mappings: mappings})
+      [CommentThread, Comment].each do |model|
```

Comment:
Hey @maxrothman. I thought the change of order of these mappings (models) was benign, but it turns out that this was the source of the error when calling put_mapping in a separate API call.  I had to reverse what was "Comment, CommentThread".

Now, I'm not sure why the order of these matter, and whether this is all fine or indicative of some other bug.